### ISSUE TITLE
Add TLS status and improved pending intent handling

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#68]
```
bundle includes:
-updated german translation
-common drawable resource directory for shared drawables
-added TLS status TextView to MainScreen
-only create pending intents and a sms object if they are needed

(sorry, accidentally deleted to much drawables, all is back to good with the 
last changeset)
```

Original issue reported on code.google.com by `fschm...@gmail.com` on 30 Jan 2011 at 3:03

Attachments:
- [tls_status_and_pending_intent_handling.bundle](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-68/comment-0/tls_status_and_pending_intent_handling.bundle)

**Comment by GoogleCodeExporter** [comment]
```
I'll replace the TLS label by a lock icon.
You set 50px instead of 30px for the status bar height so I set 40px :)

```

Original comment by `Florent....@gmail.com` on 30 Jan 2011 at 4:01

**Comment by GoogleCodeExporter** [comment]
```
Could find a good looking icon in the first place, but I didn't do an excessive 
search  either. :)
```

Original comment by `fschm...@gmail.com` on 30 Jan 2011 at 4:30

**Comment by GoogleCodeExporter** [comment]
```
I've put back XmppManager.broadcastStatus as static for home widget update 
purpose when _xmpp instance is null.
Android framework contains a lot of icons.
```

Original comment by `Florent....@gmail.com` on 30 Jan 2011 at 4:32

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 30 Jan 2011 at 4:41
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.9
```

Original comment by `Florent....@gmail.com` on 6 Feb 2011 at 8:27
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
